File: Assets/Script/Main_Menu.js
var quit=false;


function OnMouseEnter()
{
	GUI.color = Color.red;
	//renderer.material.color = Color.red;		//Change Color to red!
	//CanvasRenderer.SetColor(Color.red);
}

function OnMouseExit()
{
	//renderer.material.color = Color.white;		//Change Color to white!

}

function OnMouseUp()
{

	if (quit == true)
	{
		Application.Quit();					    //If you click on quit aplication quits.
	}
	else
	{
		Application.LoadLevel(1);				//If you click on other button it loads game!
	}

}

File: Assets/Script/MainMenu.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class MainMenu : MonoBehaviour {

	public bool isQuit = false;
	public bool isContinue = false;

	private GameObject informationHandler;
	private InformationHandler infoHandler;
	private Text text;

	void Awake() {
		informationHandler = GameObject.Find ("InformationHandler");
		infoHandler = informationHandler.GetComponent<InformationHandler> ();
	}

	// Use this for initialization
	void Start () {
		text = GetComponent<Text> ();
	}

	public void OnMouseEnter()
	{
		text.color = Color.red;
	}
	
	public void OnMouseExit()
	{
		text.color = Color.white;
	}
	
	public void OnMouseUp()
	{
		
		if (isQuit) {
			Application.Quit ();					    //If you click on quit aplication quits.
		} else if (isContinue) {
			informationHandler.SetActive(true);
			Application.LoadLevel(2);				//If you click on other button it loads game!
		}else {
			infoHandler.updateInformationData(2);
			informationHandler.SetActive(true);
			Application.LoadLevel(2);				//If you click on other button it loads game!
		}
		
	}

	void Update() {

	}
}
